packages/lodestar/src/sync/options.ts

```diff
@@ -9,7 +9,7 @@ export interface ISyncOptions {
 }
 
 export const defaultSyncOptions: Required<ISyncOptions> = {
-  minPeers: 2,
+  minPeers: 4,
```

Comment:
just looking for a little bit of background here....where did we get our initial number of 2 for minPeers and how do we know that increasing it to 4 won't have any adverse side-effects?

Reply:
this is the minimal number of peers sync wait to start its process. If we start from scratch we can quickly get ~20 peers. As I remember, initially we had a hard time getting peers, that's quite it's quite low.

With the 2 peers minimal, the common finalized checkpoint may be too small if the found peers are syncing peer too (this may happen if we're unlucky). The consequence is we use Regular Sync instead of Initial Sync even there are a lot of finalized epochs ahead bc it bypass initial sync unintentionally.

https://github.com/ChainSafe/lodestar/blob/master/packages/lodestar/src/sync/initial/fast/index.ts#L78

Increasing to 4 peers help me fix the issue locally, we can even increase more. It has no side-effect.

Reply:
what value does other clients have?

Reply:
it's 3 for Prysm, not sure for Lighthouse